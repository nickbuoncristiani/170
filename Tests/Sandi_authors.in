86
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86
70 58 26 41 37 49 59 53 18 84 38 52 74 65 60 50 69 51 79 31 10 17 13 29 8
25
x 1.3994 1.16115 x x x x x x x x x x x x x x x x x x x x x x x x x x 1.95058 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.13447 x x x x x x x x x x x x x x x
1.3994 x 1.33189 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.16115 1.33189 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.16695 x 1.44822 x 1.11861 1.71664 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.16695 x 1.93292 1.86883 1.12475 1.46277 1.73583 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.93292 x x 1.21423 x x x x x x x x x x x x x x x x x x x x x 1.53075 x x x x x x x x x x x x x x x x x x x x x x x x x 1.6607 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.44822 1.86883 x x x 1.29451 1.30541 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x 1.12475 1.21423 x x x x x x x x x 1.6778 x x x x x x x x x x x x x 1.16897 x x x x x x x 1.73131 x x x x x x x x x x x x x x x 1.7114 x 1.92585 x x x x x x x x x x x 1.5522 1.34171 1.39941 x x x x x x x x x x x x x x x x
x x x 1.11861 1.46277 x 1.29451 x x 1.83817 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.74655 1.92425 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x 1.71664 1.73583 x 1.30541 x 1.83817 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.65243 x x x x x x x x x x x x x x x x x 1.90029 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x 1.65243 x 1.58891 1.79606 1.88312 1.84732 1.36414 1.16284 1.92388 1.38128 1.45996 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.58891 x x x x x x 1.25383 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.79606 x x 1.51274 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.88312 x 1.51274 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.5708 x x x x x x x x x 1.4724 1.28778 1.04866 1.59993 1.70762 x x 1.14606 1.98257 1.20974 1.98869 x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.6778 x x x 1.84732 x x x x x x x x 1.80212 x x x x x x x x 1.20789 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.21312 x x x x x x x x x x x x
x x x x x x x x x x x 1.36414 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.16284 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.92388 1.25383 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.38128 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.45996 x x x 1.80212 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x 1.49911 x x x x x x x x x x x x 1.9354 1.1417 x x x x x x x x x x x x x x x 1.99741 x x x x x x x x x x x x x x x x x x x x x x x x x 1.97163 1.33461 1.02393 x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.49911 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x 1.34942 1.10405 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83764 x x x x x x x x x x x x x x x x x 1.89282 x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.34942 x 1.01845 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.10405 1.01845 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.71106 x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.34555 1.5228 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.34555 x 1.68075 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x 1.5228 1.68075 x x 1.84441 x x x x x x x x x x x x x x x x x 1.78788 x x 1.10919 x x 1.84271 x x x x x x x x x x x x x 1.98653 x x x x x x x x 1.04803 x x x x x x x x
1.95058 x x x x 1.53075 x 1.16897 x x 1.90029 x x x x 1.20789 x x x x x x x x x x x x x x x x x 1.81245 x x x x x x x x x x x x x x x x x x 1.47157 x x x x x x x x x x x x x x x x x x 1.45989 1.65805 1.65805 1.4149 x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.84441 x x 1.05757 1.59473 x x x x x x x x x x x x x x x 1.5087 1.15578 1.7245 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.05757 x 1.04633 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.59473 1.04633 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.15306 x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.81245 x x x x 1.54781 x x x x x x x x x x x x x x x x x 1.23031 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.54781 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.9354 x x x x x x x x x x x x x x 1.72253 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.1417 x x x x x x x x x x x x x 1.72253 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.73131 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.28536 x x x x x x x x x x x x x x x x x x x x 1.68527 x x x x x x x 1.51088 x x x x x x x x 1.1224 x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.28536 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.12575 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.12575 x 1.96546 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.96546 x x x x 1.64867 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.53367 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.74655 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.53367 x 1.60937 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x 1.92425 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.60937 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.5708 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.64867 x x x x 1.93403 x x x x x x x x x x x x x x x x x x 1.13034 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.93403 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.78788 x 1.5087 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.15578 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.7245 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.61431 x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.10919 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.99741 x x x x x x x 1.47157 x x x 1.23031 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.41811 1.8415 1.30053 x x x x x
x x x x x x x 1.7114 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.07268 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.84271 x x x x x x x x x x x x x x x x x x x x x x x x 1.07268 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x 1.6607 x 1.92585 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.4724 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.07889 1.83088 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.28778 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.07889 x 1.74464 x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.04866 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.83088 1.74464 x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.59993 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.05202 1.02096 x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.70762 x x x x x x x x x x x x x x x x x x x x x x x 1.68527 x x x x x x x x x x x x x x x x x x x x 1.05202 x x x x x x x x 1.77849 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.02096 x x x x x x x x x x x x x x x 1.2662 1.2532 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x 1.71106 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.14606 x x x x x x x x 1.83764 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.98257 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.20974 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.55363 x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x 1.98869 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.13034 x x x x x x x x x x x x x x x x x x 1.55363 x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.5522 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x 1.34171 x x x x x x x x x x x x x x x x x x x x 1.98653 x x x x x x x x x 1.51088 x x x x x x x x x x x x x x x x x x x x x 1.77849 x x x x x x x x x x x x x x x x 1.11129 x x x x x x x x
x x x x x x x 1.39941 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.9786 x x 1.57705
1.13447 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.45989 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.65805 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x 1.21312 x x x x x x x x x x x x x 1.65805 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.4149 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.2662 x x x x x x x x x x x x x x 1.09283 x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.2532 x x x x x x x x x x x x x 1.09283 x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.04803 x x x x x x x x x 1.1224 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.11129 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.97163 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.41811 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.33461 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.8415 x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.06476 x x x x x
x x x x x x x x x x x x x x x x x x x x x 1.02393 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.30053 x x x x x x x x x x x x x x x x x x x x x x x x x x 1.06476 x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x 1.89282 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.9786 x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.61431 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.15306 x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x x 1.57705 x x x x x x x x x x x x x x x x
