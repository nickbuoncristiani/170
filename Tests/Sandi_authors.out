25 26 24 64 15 60 61 69 8 69 61 60 15 61 69 29 31 51 31 49 29 69 61 15 58 59 15 64 24 25
11
26 26
15 65 41
60 60
69 69
8 17 18 70 38 8 13
29 52 29
31 50 31
51 51 84
49 49
58 58
59 59
